.globalconfig

```diff
@@ -0,0 +1,18 @@
+is_global = true
+
+# Globalization rules
+dotnet_code_quality.CA1305.excluded_symbol_names = T:System.Byte|T:System.SByte|T:System.Int16|T:System.UInt16|T:System.Int32|T:System.UInt32|T:System.Int64|T:System.UInt64|T:System.String|T:System.Text.StringBuilder|T:System.Convert
+
+dotnet_diagnostic.CA1305.severity = error
+dotnet_diagnostic.CA2101.severity = suggestion
+
+# Performance rules
+dotnet_code_quality.CA1826.exclude_ordefault_methods = true
+
+dotnet_diagnostic.CA1805.severity = silent
+dotnet_diagnostic.CA1822.severity = silent
+dotnet_diagnostic.CA1838.severity = suggestion
+
+# Usage rules
+dotnet_diagnostic.CA1816.severity = none
```

Comment:
I agree on leaving this off for now because it hits a ton of lines and looks possibly complex to refactor around, but would prefer a # TODO to turn it on at some point

Reply:
CA1816 is pretty useless imo, it's a nudge to call a method on dispose that is ONLY relevant when:
1. The class has been/will be/could be derived by anything at any point
2. The class itself does not implement a finalizer, but the derived class does. It may also only be relevent when that derived class itself doesn't call `GC.SuppressFinalize`, however I'm not sure about that

In any case, this is relevant for 0% of BizHawk code imho.